source/features/clean-conversation-sidebar.tsx

```diff
@@ -66,7 +66,8 @@ async function clean(): Promise<void> {
 
 	// Assignees
 	const assignees = select('.js-issue-assignees')!;
-	if (assignees.children.length === 0) {
+	// `#assignees-select-menu` check is for collaborators with only "triage access" #4238
+	if (assignees.children.length === 0 && !select('#assignees-select-menu')) {
```

Comment:
What I meant in my previous comment is that `#assignees-select-menu` covers both cases, so `assignees.children.length === 0` is a redundant check.

The same exact thing happened in https://github.com/sindresorhus/refined-github/pull/4336#discussion_r628903336

Here's a comparison for history purposes:

## Full access

<img width="300" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/1402241/117604479-b32d8580-b17f-11eb-82ac-e946ad2b6582.png">

## Triage access

![114912044-8a6ad800-9ded-11eb-8540-fdef5d868a88](https://user-images.githubusercontent.com/1402241/117604490-b759a300-b17f-11eb-85f3-50c6694b40ac.png)